include/9on12Device.h

```diff
@@ -90,7 +90,8 @@ namespace D3D9on12
         {
             const UINT CONSTANT_BUFFER_ELEMENT_SIZE = sizeof(FLOAT) * 4;
             UINT firstConstant = offsetInBytes / CONSTANT_BUFFER_ELEMENT_SIZE;
-            GetContext().SetConstantBuffers<ShaderStage>(shaderRegister, 1, &pResource, &firstConstant, nullptr);
+            UINT maxConstantBufferElements = ConstantsManager::g_cMaxConstantBufferSize / sizeof(FLOAT);
+            GetContext().SetConstantBuffers<ShaderStage>(shaderRegister, 1, &pResource, &firstConstant, (UINT *)&maxConstantBufferElements);

```

Comment:
`&maxConstantBufferElements` is already a `UINT *`, I don't see why you'd need the cast?